bit/network/services.py

```diff
@@ -357,6 +414,42 @@ def get_transactions_testnet(cls, address):
 
         raise ConnectionError('All APIs are unreachable.')
 
+    @classmethod
+    def get_transaction_by_id(cls, txid):
+        """Gets a transaction by its transaction id (txid).
+
+        :param txid: The id of the transaction
+        :type txid: ``str``
+        :raises ConnectionError: If all API services fail.
+        :rtype: ``TxObj``
+        """
+
+        for api_call in cls.GET_TRANSACTION_BY_ID_MAIN:
+            try:
+                return deserialize(api_call(txid))
```

Comment:
On second thought, maybe it is better to simply return the transaction as hex and then if needed deserialize it on the other end where it is being used? It feels more concise this way and we also prevent deserializing and then reserializing again if the hex is needed (e.g. for PSBT).

Reply:
I personally like the code to `deserialize` before return to save a step that you're almost always going to do anyways. That being said, I'm not against this change if you feel it better fits the library as a whole.

Please let me know the final decision and I can make the change as requested.

Reply:
I am working on implementing PSBT to Bit where an API function is needed that returns the raw serialized hex-encoded transaction. Therefore I prefer the function `get_transaction_by_id` to return just that instead of an object which would have to be serialized again. I also prefer it since the other functions inside NetworkAPI also return simple hex values. 

So if nobody has any objection against it I would like that change to be implemented. :)

Reply:
I've made the change as requested. I accidentally skimmed over your point about PSBT in the first post, but the use case makes sense. I'm glad this will potentially get some use other than in my case.